### ISSUE TITLE
ptune训练后怎么推理[BUG/Help] <title>

### ISSUE THREAD

**Issue by liaoweiguo** [THUDM/ChatGLM-6B#396]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

看不到推理效果，是需要在输入做什么处理吗？模型输入在是简单的文本，还是需要加上“content: ”什么的

>>> model.chat(tokenizer,promps[1])
The dtype of attention mask (torch.int64) is not bool
('简约的无袖鱼尾裙，简约的条纹设计，简单大方。无袖的设计，修饰手臂线条，凸显锁骨。裙身撞色的条纹，简洁大方。', [('类型#裙*风格#简约*图案#条纹*图案#线条*图案#撞色*裙型#鱼尾裙*裙袖长#无袖', '简约的无袖鱼尾裙，简约的条纹设计，简单大方。无袖的设计，修饰手臂线条，凸显锁骨。裙身撞色的条纹，简洁大方。')])
>>> model.chat(tokenizer,promps[100])
('时尚休闲的短裤，搭配舒适舒适的材质，带来舒适的穿着体验。休闲的设计，穿着舒适自然，轻松穿出个性时尚。舒适的面料，穿着舒适不扎，带来舒适的感觉。', [('类型#裤*风格#休闲*裤长#短裤', '时尚休闲的短裤，搭配舒适舒适的材质，带来舒适的穿着体验。休闲的设计，穿着舒适自然，轻松穿出个性时尚。舒适的面料，穿着舒适不扎，带来舒适的感觉。')])
>>> model.chat(tokenizer,f"content: {promps[100]}")
('休闲风的短裤，整体采用浅蓝色的色调，在视觉上给人一种清新干净的感觉，并且穿起来也会比较舒适，上身也很显气质。而裤脚采用小开叉的设计，在修饰腿型的同时，也能让整体看起来比较精致。', [('content: 类型#裤*风格#休闲*裤长#短裤', '休闲风的短裤，整体采用浅蓝色的色调，在视觉上给人一种清新干净的感觉，并且穿起来也会比较舒适，上身也很显气质。而裤脚采用小开叉的设计，在修饰腿型的同时，也能让整体看起来比较精致。')])
>>> model.chat(tokenizer,f"content: {promps[0]}")
('宽松版型的阔腿裤，穿上身十分随性，彰显个性魅力，彰显女性性感迷人的曲线。裤腿两侧采用了收束的设计，让整条裤子更加流畅自然，上身更有型。', [('content: 类型#裤*版型#宽松*风格#性感*图案#线条*裤型#阔腿裤', '宽松版型的阔腿裤，穿上身十分随性，彰显个性魅力，彰显女性性感迷人的曲线。裤腿两侧采用了收束的设计，让整条裤子更加流畅自然，上身更有型。')])
>>> model.chat(tokenizer,f"content: {promps[0]}\n summary")
('宽松的阔腿裤设计，上身更显高挑，穿着舒适，上身性感迷人。宽松的版型，不挑身材，上身穿着不勒肉，穿着舒适。线条流畅，修饰腿部线条，穿着更显高挑。', [('content: 类型#裤*版型#宽松*风格#性感*图案#线条*裤型#阔腿裤\n summary', '宽松的阔腿裤设计，上身更显高挑，穿着舒适，上身性感迷人。宽松的版型，不挑身材，上身穿着不勒肉，穿着舒适。线条流畅，修饰腿部线条，穿着更显高挑。')])

### Steps To Reproduce

1. 按ptuning训练
2. 推理，如下：

modelname = "ptuning/output/adgen-chatglm-6b-pt-8-1e-2/checkpoint-3000"
tokenizer = AutoTokenizer.from_pretrained(modelname, trust_remote_code=True)
model = AutoModel.from_pretrained(modelname, trust_remote_code=True).half().cuda()
model = model.eval()

promps = []
with open("ptuning/AdvertiseGen/train.json") as fp:
    promps = [json.loads(p)['content'] for p in fp.readlines()]

model.chat(tokenizer,promps[1])
model.chat(tokenizer,f"content: {promps[0]}")

### Environment

```markdown
- OS: Ubuntu2004
- Python: 3.8
- Transformers: 4.27.1
- PyTorch: 
- CUDA Support (`python -c "import torch; print(torch.cuda.is_available())"`) :
```

**Comment by duzx16** [comment]
不需要加前缀，第一条的推理结果就是正确的

**Comment by liaoweiguo** [comment]
> 不需要加前缀，第一条的推理结果就是正确的

第一条结果跟没调一样的
